packages/lib/getEventTypeById.ts

```diff
@@ -227,7 +227,7 @@ export default async function getEventTypeById({
       app: {
         enabled: true,
         categories: {
-          hasSome: [AppCategories.conferencing, AppCategories.video],
+          hasSome: [AppCategories.conferencing, AppCategories.video, AppCategories.payment],
```

Comment:
While saving any payment app since payment category wasn't included here it will save default stripe app metadata.

Reply:
It can be confusing but I think eventType.price and currency should only live there and not on metadata. I can confirm that both value should have the correct one.